ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_seviri.py

```diff
@@ -0,0 +1,441 @@
+            satid2 = satid[mask]
+            satzenang2 = satzenang[mask]
+            chanfreq2 = chanfreq[mask]
+            obstype2 = obstype[mask]
+            pressure2 = pressure[mask]
+            height2 = height[mask]
+            stnelev2 = stnelev[mask]
+
+            # Processing Center
+            ogce2 = ogce[mask]
+
+            # QC Info
+            qifn2 = qifn[mask]
+
+            # Method
+            swcm2 = swcm[mask]
+
+            # ObsValue
+            wdir2 = wdir[mask]
+            wspd2 = wspd[mask]
+            uob2 = uob[mask]
+            vob2 = vob[mask]
+
+            # Timestamp Range
+            timestamp2_min = datetime.fromtimestamp(timestamp2.min())
+            timestamp2_max = datetime.fromtimestamp(timestamp2.max())
+
+            # Check unique observation time
+            unique_timestamp2 = np.unique(timestamp2)
+            logger.debug(f'Processing output for satid {sat}')
+
+            # Create the dimensions
+            dims = {
+                'Location': np.arange(0, wdir2.shape[0])
+            }
+
+            # Create IODA ObsSpace
+            iodafile = f"{cycle_type}.t{hh}z.{data_type}.{satinst}.tm00.nc"
+            OUTPUT_PATH = os.path.join(ioda_dir, iodafile)
+            logger.info(f'Create output file : {OUTPUT_PATH}')
+            obsspace = ioda_ospace.ObsSpace(OUTPUT_PATH, mode='w', dim_dict=dims)
+
+            # Create Global attributes
+            logger.debug('Write global attributes')
+            obsspace.write_attr('Converter', converter)
+            obsspace.write_attr('sourceFiles', bufrfile)
+            obsspace.write_attr('dataProviderOrigin', data_provider)
+            obsspace.write_attr('description', data_description)
+            obsspace.write_attr('datetimeReference', reference_time)
+            obsspace.write_attr('datetimeRange', [str(timestamp2_min), str(timestamp2_max)])
+            obsspace.write_attr('sensor', sensor_id)
+            obsspace.write_attr('platform', satellite_id)
+            obsspace.write_attr('platformCommonName', satellite_name)
+            obsspace.write_attr('sensorCommonName', sensor_name)
+            obsspace.write_attr('processingLevel', process_level)
+            obsspace.write_attr('platformLongDescription', platform_description)
+            obsspace.write_attr('sensorLongDescription', sensor_description)
+
+            # Create IODA variables
+            logger.debug('Write variables: name, type, units, and attributes')
+            # Longitude
+            obsspace.create_var('MetaData/longitude', dtype=lon2.dtype, fillval=lon2.fill_value) \
+                .write_attr('units', 'degrees_east') \
+                .write_attr('valid_range', np.array([-180, 180], dtype=np.float32)) \
+                .write_attr('long_name', 'Longitude') \
+                .write_data(lon2)
+
+            # Latitude
+            obsspace.create_var('MetaData/latitude', dtype=lat.dtype, fillval=lat2.fill_value) \
```

Comment:
(not that it should matter, but more for consistency)

Reply:
@nicholasesposito I think they are functionally equivalent since `lon2 = lon[mask]`, but for the sake of consistency it's probably a good idea to change it to `lat2.dtype`

Reply:
resolved in [ca02888](https://github.com/NOAA-EMC/GDASApp/pull/764/commits/ca028881aa72fd1276f223f92ad0fb6bff1af709)

Reply:
Great. Thanks!